janitor/xarray/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .functions import *
```

Comment:
Just a small comment suggestion here, so my brain doesn't get triggered that badly by `import *`.

```suggestion
"""Import all functions and utils into xarray submodule namespace."""
from .functions import *
```